modules/tibc/core/types/genesis.go

```diff
@@ -28,6 +30,9 @@ func (gs *GenesisState) Validate() error {
 	if err := gs.ClientGenesis.Validate(); err != nil {
 		return err
 	}
+	if err := gs.PacketGenesis.Validate(); err != nil {
+		return err
+	}
 
-	return gs.PacketGenesis.Validate()
+	return nil
```

Comment:
What's the difference?

Reply:
For the next update of routing's validate

Reply:
Why not do it in the next update?